tests/unittests/algo/nevergrad/requirements.txt

```diff
@@ -0,0 +1,9 @@
+## Python requirements for running tests.
+##
+##    pip install --requirement=requirements.txt
+##
+pytest>=3.5
```

Comment:
Ideally these requirements would be in the algo extra dependencies in setup.py or in test extra dependencies in setup.py.